src/Native/jitinterface/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(jitinterface)
+
+set(NATIVE_SOURCES
+    jitinterface.cpp
+    jitwrapper.cpp
+    corinfoexception.cpp
+)
+
+if (WIN32)
+    list(APPEND NATIVE_SOURCES jitinterface.def)
+    add_compile_options(/Zi)
```

Comment:
This should be in the global CMakeFile one level up (same as in CoreCLR). We want to have debug symbols for every library that we build.